x-pack/metricbeat/module/aws/ec2/ec2.go

```diff
@@ -103,19 +107,27 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 	}
 
 	awsConfig.Region = moduleConfig.DefaultRegion
-
 	svcEC2 := ec2.New(awsConfig)
 	regionsList, err := getRegions(svcEC2)
 	if err != nil {
 		err = errors.Wrap(err, "getRegions failed")
-		logp.Error(err)
+		ec2Logger.Errorf(err.Error())
+	}
+
+	//Calculate duration based on period
+	durationString, periodSec, err := convertPeriodToDuration(moduleConfig.Period)
```

Comment:
This would be a good place to log once if we see a period that is not a multiple of 60s or 300s?

Reply:
yes! Thanks! I forgot that piece...